### PR TITLE
Fix extract templates examples

### PR DESCRIPTION
**Pull request by panosk**

### CODE REVIEW

**Comment by maxbachmann** [comment]
Thanks for fixing the example. Could you remove my previous commit from this PR? I mistakenly pushed this on main yesterday.

File: README.md [hunk 1]
```diff
@@ -159,7 +159,7 @@ extract(const Sentence1& query, const Iterable& choices, const double score_cuto
 {
   std::vector<std::pair<Sentence2, double>> results;
 
-  auto scorer = rapidfuzz::fuzz::CachedRatio<Sentence1>(query);
+  auto scorer = rapidfuzz::fuzz::CachedRatio<typename Sentence1::value_type>(query);
```
Comment (maxbachmann) [inline README.md:162]: I guess this could be simplified to
```
rapidfuzz::fuzz::CachedRatio<typename Sentence1::value_type> scorer(query);
```

Comment (panosk) [reply README.md]: Sure, done!

**Comment by maxbachmann** [comment]
Thanks for the contribution

**Pull request merged**